src/main/java/org/sonar/plugins/resharper/ReSharperReportParser.java

```diff
@@ -62,8 +66,11 @@
           if (stream.next() == XMLStreamConstants.START_ELEMENT) {
             String tagName = stream.getLocalName();
 
-            if ("Issue".equals(tagName)) {
-              handleIssueTag();
+            if ("Project".equals(tagName)) {
+              if(handleProjectTag()) {
+                // break after project tag has been handled.
+                break;
```

Comment:
In Visual studio, I can't rename my a project to have the same name as another project. The only (imaginable) scenario would be when you tell the inspectcode tool, to inspect multiple projects instead of your entire solution.